### PR TITLE
Edit presets dialogue

### PR DESCRIPTION
**Pull request by Markvanderuit**
- Shots now have an edit preset dialogue.
- Shot toast now displays preset content.
- Stuff still needs styling.

### CODE REVIEW

**Comment by bart-de-jonge** [comment]
Please don't push so many failing builds, chech offline

File: elements/timeline-view/timeline-dialog.html [hunk 1]
```diff
@@ -30,11 +61,68 @@
...
+        }
+
+        /**
+         * Respond to set event
+         */
+        handleSetPresetsResponse(event) {
+            if(event.detail.response.success) {
+                this.block_data.presetId = this.preset.id;
+            }
+            console.log(event.detail.response);
```
Comment (bart-de-jonge) [inline elements/timeline-view/timeline-dialog.html]: please remove this log statement

File: elements/timeline-view/timeline-grid.js [hunk 2]
```diff
@@ -91,7 +91,8 @@ class timelineGrid {
         newData.minCount = 0;
         newData.maxCount = maxCount;
         newData.cameraTimelines = [];
-        
+
+
```
Comment (bart-de-jonge) [inline elements/timeline-view/timeline-grid.js]: whitespace

**Comment by mennoo1996** [comment]
Remove those log statements!. For the rest, great work

**Comment by bart-de-jonge** [comment]
2 small comments, please take a look

**Pull request merged**